docs-src/modules/getting-started/pages/hello-world-cli.adoc

```diff
@@ -243,7 +230,7 @@ Here's a more sophisticated example that touches on many of the unique features
 let cadence = hourly()
 
 # Anything can be named and re-used
-let hourly_big_purchases = purchase
+let hourly_big_purchases = Purchase
```

Comment:
the query as written still errors after this change.  I think we should update it to something that doesn't error, but lets do that in a different PR.

```
state: STATE_FAILURE
config:
  dataTokenId: 05f3eeb3-5d7d-4cb9-8d25-8e8ec6b0a85a
analysis: {}
fenlDiagnostics:
  fenlDiagnostics:
  - severity: SEVERITY_ERROR
    code: E0006
    message: Unbound reference
    formatted: |+
      error[E0006]: Unbound reference
         --> Query:23:27
         |
      23 |   last_visit_region: last(Pageview.region)
         |                           ^^^^^^^^ No reference named 'Pageview'
         |
         = Nearest matches: 'cadence', 'Purchase', '$input', 'purchases_now', 'purchases_yesterday'

  numErrors: "1"
metrics: {}
requestDetails:
  requestId: 0ba4b3980e0d372703eca40e69c8fa45
```